utils/build/docker/set-system-tests-weblog-env.Dockerfile

```diff
@@ -20,6 +20,7 @@ ENV DD_IAST_ENABLED=true
 ENV DD_IAST_REQUEST_SAMPLING=100
 ENV DD_IAST_MAX_CONCURRENT_REQUESTS=10
 ENV DD_IAST_DEBUG_ENABLED=true
+ENV DD_IAST_REDACTION_ENABLED=false
```

Comment:
We could use a comment explaining why this is here.

Reply:
Done :smile: